Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -523,18 +556,26 @@ namespace AZ
         AZStd::any MaterialComponentController::GetPropertyOverride(const MaterialAssignmentId& materialAssignmentId, const AZStd::string& propertyName) const
```

Comment:
I agree as you stated in the description, we should probably rename this to GetPropertyValue. Same for the script canvas node. A separate PR is fine IMO.